src/components/ebay-toggle-button/toggle-button.stories.js

```diff
@@ -0,0 +1,162 @@
+});
+
+export default {
+    title: "buttons/ebay-toggle-button",
+    component,
+    parameters: {
+        docs: {
+            description: {
+                component: readme,
+            },
+        },
+    },
+    argTypes: {
+        renderBody: {},
+        layoutType: {
+            type: "string",
+            control: { type: "select" },
+            options: ["minimal", "list", "gallery"],
+            description:
+                'Enforced layout type of the button. May be `"minimal"` (default), `"list"`, or `"gallery"`',
+        },
+        pressed: {
+            type: "boolean",
+            control: { type: "boolean" },
+            description: "Pressed state of the button",
+        },
+        title: {
+            type: "string",
+            control: { type: "text" },
+            description: "Title attribute for the button",
+        },
+        subtitle: {
+            type: "string|@subtitle",
+            control: { type: "text" },
+            description: "Subtitle attribute for the button",
+        },
+        icon: {
+            name: "@icon",
+            description: "An `<ebay-[name]-icon>` to show as the button's icon",
+            table: {
+                category: "@attribute tags",
+            },
+        },
+        img: {
+            name: "@img",
+            description: "An `<img>` to show as the button's image",
+            table: {
+                category: "@attribute tags",
+            },
+        },
+        subtitleTag: {
+            name: "@subtitle",
+            description:
+                "May be used instead of the `subtitle` attribute for more control. Should contain no more than two brief lines of text",
+            table: {
+                category: "@attribute tags",
+            },
+        },
+        src: {
+            table: {
+                category: "@img attributes",
+            },
+            control: { type: "text" },
+            description: "Link to the image source",
+        },
+        alt: {
+            table: {
+                category: "@img attributes",
+            },
+            control: { type: "text" },
+            description: "Alt text for the image",
+        },
+        fillPlacement: {
+            table: {
+                category: "@img attributes",
+            },
+            control: { type: "text" },
+            description:
+                "Placement of the image within the given bounds using the CSS `background-position` property. Using this property will switch the image fit from `contain` to `cover`",
+        },
+        onToggle: {
+            action: "on-toggle",
+            description: "Triggered when the button is toggled",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "{ originalEvent, pressed }",
+                },
+            },
+        },
+    },
+};
+
+export const Default = Template.bind({});
+Default.args = {
```

Comment:
The default/minimal layout can't have subtitles, actually. Let's remove it here.